src/main/resources/application.conf

```diff
@@ -43,6 +43,9 @@ backend {
   //  baseExecutionBucket = ""
   //  endpointUrl = ""
   //}
+  local {
+    tryToHardlinkInputs = true
+  }
```

Comment:
`tryToHardlinkInputs` should also be valid for the 'sge' backend.  To avoid copying these values, maybe we should have a section about how the local-filesystem behaves:

```hocon
backend {
  local-filesystem {
    root: "cromwell-executions"
    localization: [
      "hard-link", "soft-link", "copy" // ordered values.  If one fails, try the next.  Ignore certain values in certain contexts (e.g. don't try to soft-link when using Docker)
    ]
  }
}
```

We can do this on another branch though